apps/util/lib/util.ex

```diff
@@ -25,6 +27,15 @@ defmodule Util do
     now() |> Timex.to_date()
   end
 
+  @spec time_is_greater_or_equal(DateTime.t(), DateTime.t()) :: boolean
+  def time_is_greater_or_equal(time, ref_time) do
```

Comment:
This is nice. Could you please add a `?` to the end of the function name to match [Elixir naming conventions](https://hexdocs.pm/elixir/master/naming-conventions.html#trailing-question-mark-foo)?